$NetBSD: patch-sapi_fpm_php-fpm.conf.in,v 1.1 2013/02/28 21:43:04 imil Exp $

Uncomment the pid parameter so the pidfile is created and rc.d script
can really start / stop php-fpm

--- sapi/fpm/php-fpm.conf.in.orig	2013-01-16 07:10:30.000000000 +0000
+++ sapi/fpm/php-fpm.conf.in
@@ -22,7 +22,7 @@
 ; Pid file
 ; Note: the default prefix is @EXPANDED_LOCALSTATEDIR@
 ; Default Value: none
-;pid = run/php-fpm.pid
+pid = run/php-fpm.pid
 
 ; Error log file
 ; If it's set to "syslog", log is sent to syslogd instead of being written
